src/main/java/org/jabref/logic/util/io/BackupFileUtil.java

```diff
@@ -66,18 +67,12 @@ public static Path getPathForNewBackupFileAndCreateDirectory(Path targetFile, Ba
         return directory.resolve(fileName);
     }
 
-    /**
-     * Finds the latest backup (.sav). If it does not exist, an empty optional is returned
-     *
-     * @param targetFile the full path of the file to backup
-     */
-    public static Optional<Path> getPathOfLatestExistingBackupFile(Path targetFile, BackupFileType fileType) {
+    public static Optional<Path> getPathOfLatestExistingBackupFile(Path targetFile, BackupFileType fileType, Path backupDir) {
         // The code is similar to "getPathForNewBackupFileAndCreateDirectory"
 
         String extension = "." + fileType.getExtensions().get(0);
 
-        Path directory = getAppDataBackupDir();
```

Comment:
nitpick: getAppDataBackupDir could be renamed to getDefaultBackupDir

Reply:
That method is still valid and does return the AppDataBackupDir. However, I found another method where I did not pass the correect param